File: iOS ObjectiveC/Introduccion programación Objective C/Ejemplos/PrimerEjercicio/PrimerEjercicio/ViewController.m
//
//  ViewController.m
//  PrimerEjercicio
//
//  Created by Julio Laptop on 11/18/16.
//  Copyright © 2016 Julio Galeano. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@property (nonatomic, assign) NSInteger conteo;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    self.conteo = 0;
    // Do any additional setup after loading the view, typically from a nib.
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (IBAction)clickBoton:(UIButton *)sender {
    self.conteo++;
    NSString *mensaje =  @"Click en el boton: ";
    self.texto.text = [mensaje stringByAppendingString:[NSString stringWithFormat:@"%d", (int)self.conteo]];
}

@end

File: Swift/PruebasUI/PruebasUI/ViewController.swift
//
//  ViewController.swift
//  PruebasUI
//
//  Created by Julio Cesar Galeano on 10/10/19.
//  Copyright © 2019 Julio Cesar Galeano. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var textLabel: UILabel!
    @IBOutlet weak var cajaTexto: UITextField!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
    }

    @IBAction func cambiarTextoClick(_ sender: Any) {
        textLabel.text = "Texto cambiado"
    }
    
}

